torchci/test/jobUtils.test.ts

```diff
@@ -565,4 +569,606 @@ describe("Test various job utils", () => {
+            "test_cpp_extensions_open_device_registration.py::TestCppExtensionOpenRgistration::test_open_device_registration",
+          ],
+          name: "Anything goes",
+        },
+        [
+          {
+            state: "open",
+            number: 100152,
+            title:
+              "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+            body: "Nothing is specified here.  Does this mean the test is disabled everywhere?",
+            updated_at: "2024-05-06T00:30:00Z",
+            author_association: "",
+            html_url: "",
+          },
+        ]
+      )
+    ).toEqual([]);
+
+    // Match a disable test issue
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [
+            "test_cpp_extensions_open_device_registration.py::TestCppExtensionOpenRgistration::test_open_device_registration",
+          ],
+          name: "pull / linux-focal-py3.11-clang10 / test (default, 1, 3, linux.2xlarge)",
+        },
+        [
+          {
+            state: "open",
+            number: 100152,
+            title:
+              "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+            body: "Platforms: linux, mac",
+            updated_at: "2024-05-06T00:30:00Z",
+            author_association: "",
+            html_url: "",
+          },
+        ]
+      )
+    ).toEqual([
+      {
+        state: "open",
+        number: 100152,
+        title:
+          "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+        body: "Platforms: linux, mac",
+        updated_at: "2024-05-06T00:30:00Z",
+        author_association: "",
+        html_url: "",
+      },
+    ]);
+
+    // Include new lines in issue body
+    expect(
+      getDisabledTestIssues(
+        {
+          id: "",
+          completed_at: "",
+          html_url: "",
+          head_sha: "",
+          failure_captures: [
+            "test_cpp_extensions_open_device_registration.py::TestCppExtensionOpenRgistration::test_open_device_registration",
+          ],
+          name: "pull / linux-focal-py3.11-clang10 / test (default, 1, 3, linux.2xlarge)",
+        },
+        [
+          {
+            state: "open",
+            number: 100152,
+            title:
+              "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+            body: "Platforms: linux, mac\n\rShould not be match",
+            updated_at: "2024-05-06T00:30:00Z",
+            author_association: "",
+            html_url: "",
+          },
+        ]
+      )
+    ).toEqual([
+      {
+        state: "open",
+        number: 100152,
+        title:
+          "DISABLED test_open_device_registration (__main__.TestCppExtensionOpenRgistration)",
+        body: "Platforms: linux, mac\n\rShould not be match",
```

Comment:
why is this should not be match?

Reply:
Ah, I should explain it better.  I mean the portion of the issue body on the next line won't match as a platform.  My initial version of the regex matched with the next line after `Platforms` and that was wrong.  So, I add the test.